pages/getting-started/developers.md

```diff
@@ -116,35 +116,48 @@ You can also email us directly at uswebdesignstandards@gsa.gov.
-* Uses **modular CSS** for scalable, modular, and flexible code.
-* Uses **nesting** when appropriate. Nest minimally with up to two levels of nesting.
-* Hard-coded magic numbers are avoided and, if necessary, defined in the `core/variables` file.
-* Media queries are built **mobile first**.
-* **Spacing units** are as much as possible defined as rem or em units so they scale appropriately with text size. Pixels can be used for detail work and should not exceed 5px (For example: 3px borders).
-
-**For more information, visit:
-[https://pages.18f.gov/frontend/css-coding-styleguide/](https://pages.18f.gov/frontend/css-coding-styleguide/)**
-
-## Where things live<a id="where-things-live"></a>
-
-* **HTML** markup for the components are located in: `docs/_components` in the site root.
-* **Sass** styles are located in: `src/stylesheets/ (/core, /elements, /components)`. **Compiled CSS** is located in the [downloadable zip file]({{ site.repos[0].url }}/releases/download/v{{ site.version }}/uswds-{{ site.version }}.zip) .
-* **JS** is located in: `src/js/components (accordion.js, toggle-field-mark.js, toggle-form-input.js, validator.js)`.
-* **Fonts** are located in: `src/fonts`.
-* **Images** and icons are located in: `src/img`.
-
-## Notes on accessibility<a id="notes-on-accessibility"></a>
-
-We’ve designed the Standards to support older and newer browsers through progressive enhancement, and they officially support Internet Explorer 9 and up, along with the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage, and most agency websites should be able to safely begin support at Internet Explorer 9.
-
-The Standards also meet the [WCAG 2.0 AA accessibility guidelines](https://www.w3.org/TR/WCAG20/) and are compliant with [Section 508 of the Rehabilitation Act](http://www.section508.gov/). We’re happy to answer questions about accessibility — email us for more information.
-
-## Contribution guidelines<a id="contribution-guidelines"></a>
-
-We're so glad you're thinking about contributing to the Standards! You can find our complete [contribution guidelines](https://github.com/18F/web-design-standards/blob/staging/CONTRIBUTING.md) in our repo — please review them before submitting your contribution.
-
-If you have any questions about these guidelines (or the Standards, more generally), don’t hesitate to [email us](mailto:uswebdesignstandards@gsa.gov) — we’ll get back to you within 48 hours.
+* * Uses **[Bourbon](http://bourbon.io/)** for its simple and lightweight Sass mixin library, and the **[Neat](http://neat.bourbon.io/)** library for the grid framework. Bourbon and Neat are open-source products from **[thoughtbot](https://thoughtbot.com/)**.
```

Comment:
There are some weird formatting things going on throughout this section. What changes were we wanting to make here?